x-pack/elastic-agent/pkg/agent/program/testdata/single_config-packetbeat.yml

```diff
@@ -0,0 +1,66 @@
+packetbeat:
+  inputs:
+  - type: network/flow
+    id: network/flows-flow.network
+    index: logs-flow.network-default
+    timeout: 10s
+    period: 10s
+    keep_null: false
+    data_stream:
+      dataset: flow.network
+      type: logs
+    processors:
+      - add_fields:
+          target: 'data_stream'
+          fields:
+            type: logs
+            dataset: flow.network
+            namespace: default
+      - add_fields:
+          target: 'event'
+          fields:
+            dataset: flow.network
+      - add_fields:
+          target: 'elastic_agent'
+          fields:
+            id: agent-id
+            version: 8.0.0
+            snapshot: false
+  - type: network/icmp
+    id: network/icmp-icmp.network
+    index: logs-icmp.network-default
```

Comment:
I think this validates my suspicion that network is more generic. In general the indexing strategy tries to go from the left, very generic to more specific on the right. So I would turn this around to `logs-network.icmp-*`

Reply:
Sorry, I'll update that, I just copied over some policy configurations from a PoC integration package I threw together that has a bad folder structure.